src/main/resources/assets/js/components/AppComponent.jsx

```diff
@@ -36,6 +78,9 @@ var AppComponent = React.createClass({
             {model.formatTasksRunning()}
           </span> / {model.get("instances")}
         </td>
+        <td className="text-center health-bar-column">
```

Comment:
Let's replace the status column with the health column and name it "Status". It's very easy from this to see when something is deploying.